src/applications/edu-benefits/1995/config/chapters.js

```diff
@@ -27,190 +28,352 @@ const {
 
 const { educationType, serviceBefore1977 } = fullSchema1995.definitions;
 
-export const chapters = {
-  applicantInformation: {
-    title: 'Applicant Information',
-    pages: {
-      applicantInformation: createApplicantInformationPage(fullSchema1995, {
-        isVeteran: true,
-        fields: [
-          'veteranFullName',
-          'veteranSocialSecurityNumber',
-          'view:noSSN',
-          'vaFileNumber',
-        ],
-        required: ['veteranFullName'],
-      }),
-    },
-  },
-  benefitSelection: {
-    title: 'Education Benefit',
-    pages: {
-      benefitSelection: {
-        title: 'Education benefit selection',
-        path: 'benefits/eligibility',
-        uiSchema: benefitSelection.uiSchema,
-        schema: benefitSelection.schema,
-      },
-      // related to 1995-STEM
-      stem: {
-        title: 'Rogers STEM Scholarship',
-        path: 'benefits/stem',
-        uiSchema: stem.uiSchema,
-        schema: stem.schema,
-      },
-      // related to 1995-STEM
-      stemEligibility: {
-        title: 'Rogers STEM Scholarship eligibility',
-        path: 'benefits/stem-eligibility',
-        pageClass: 'vads-u-max-width--100 vads-u-vads-u-width--full',
-        depends: form => displayStemEligibility(form), // 1995-STEM related
-        uiSchema: stemEligibility.uiSchema,
-        schema: stemEligibility.schema,
-      },
-    },
-  },
-  militaryService: {
-    title: 'Military History',
-    pages: {
-      // 1995-STEM related
-      activeDuty: {
-        title: 'Active Duty',
-        path: 'active-duty',
-        depends: display1995StemFlow, // 1995-STEM related
-        uiSchema: activeDuty.uiSchema,
-        schema: activeDuty.schema,
+export const chapters = environment.isProduction()
```

Comment:
Undo this.

In addition to updating the functions `determineEligibilityFor1995Stem` and `display1995StemFlow` in `helper.jsx` to include an `environment.isProduction()` check you should just update the initial 1995-STEM page to have depends check for `environment.isProduction()`

The function `display1995StemFlow` is used to enable/disable all 1995-STEM pages and code after going through the initial page.

```
 // related to 1995-STEM
      stem: {
        title: 'Rogers STEM Scholarship',
        path: 'benefits/stem',
        depends: environment.isProduction(),
        uiSchema: stem.uiSchema,
        schema: stem.schema,
      },
```

Reply:
I see, thanks for pointing that out. Will make the change here.